src/connect/component.jsx

```diff
@@ -72,7 +86,14 @@ export const getConnectComponent = async (merchantProps) => {
       },
     });
 
-    getLogger().error("init_axo_error", { err: stringifyError(error) });
+    getLogger()
+      .track({
+        [FPTI_KEY.CONTEXT_TYPE]: "CMID",
+        [FPTI_KEY.CONTEXT_ID]: cmid,
+        [FPTI_KEY.EVENT_NAME]: `ppcp_axo_failure`,
```

Comment:
```suggestion
        [FPTI_KEY.EVENT_NAME]: `ppcp_connect_failure`,
```

Reply:
We'll also need to add `page_session_id` to all of these as well: https://github.com/paypal/paypal-sdk-constants/blob/main/src/fpti.js#L10